frontend/fingertips-frontend/lib/chartHelpers/chartHelpers.ts

```diff
@@ -12,13 +12,29 @@ export function sortHealthDataByDate(
 }
 
 export function generateSeriesData(
-  data: HealthDataForArea[]
+  data: HealthDataForArea[],
+  benchmarkData?: HealthDataForArea
 ): Highcharts.SeriesLineOptions[] {
-  return data.map((item) => ({
+  const seriesData = data.map<Highcharts.SeriesLineOptions>((item) => ({
     type: 'line',
     name: `${item.areaName}`,
     data: item.healthData.map((point) => [point.year, point.value]),
   }));
+
+  if (benchmarkData) {
+    const englandSeries: Highcharts.SeriesLineOptions = {
+      type: 'line',
+      name: 'Benchmark: England',
```

Comment:
Should this do?:
```suggestion
      name: 'Benchmark: ${benchmarkData. areaName}',
```

Given we're trying not to assume the benchmark value is England.